README.md

```diff
@@ -69,6 +69,12 @@ pub global activate dart_code_metrics
 metrics lib
 ```
 
+If you want to run the tool directly (e.g. because you are using Flutter), you can use:
```

Comment:
@creativecreatorormaybenot because it's `flutter pub` command, it seems to me that we could rename this block to `Flutter usage:`. What do you think?

Reply:
@incendial You are right, I missed that.